src/Symfony/Component/Debug/DebugClassLoader.php

```diff
@@ -173,6 +174,32 @@ public function loadClass($class)
                 @trigger_error(sprintf('The %s class is considered final%s. It may change without further notice as of its next major version. You should not extend it from %s.', $parent, self::$final[$parent], $name), E_USER_DEPRECATED);
             }
 
+            // Not an interface nor a trait
+            if (class_exists($name, false)) {
```

Comment:
In fact it's not a big deal, I can just rebase this PR if we do it in another PR.